File: app/src/main/java/com/disruption/cookcentral/widget/FavsWidgetRemoteViewsService.java
package com.disruption.cookcentral.widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class FavsWidgetRemoteViewsService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new FavsWidgetRemoteViewsFactory(getApplicationContext());
    }
}

File: app/src/main/java/com/disruption/cookcentral/widget/FavsWidgetRemoteViewsService.kt
package com.disruption.cookcentral.widget

import android.content.Intent
import android.widget.RemoteViewsService

class FavsWidgetRemoteViewsService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return FavsWidgetRemoteViewsFactory(applicationContext)
    }
}